src/app/collection/pages/collection-ncyte/footer/footer.component.html

```diff
@@ -0,0 +1,23 @@
+<section class = "footer">
+    <div class = "learn-more">
+           This is the page for the ncyte curriculum collection on 
+           CLARK (
+           <a class = "link" arial-label= "Clark Center" href="http://clark.center"> www.clark.center  </a>
+           )
+           while clark hosts the curriculumn, all rights are retained
+           by the NCyTE Center. Learn more at     
+           &nbsp;
```

Comment:
Can we try to not use these and do it via css?

Reply:
I was having an issue where the word "at" and the link were touching each other without space. This issue came up when i centered items in footer vertically. I tried to break with div too but didnt really work out. Is there spacing with css?

Reply:
You could add margin left to the a tag in css and that should solve the issue too.

Reply:
You could add margin left to the a tag in css and that should solve the issue too.